contrib/compose/README.md

```diff
@@ -0,0 +1,41 @@
+# Running dex in Docker compose
+
+This demo intends to demonstrate Dex on a development workstation. It is only intended for local test setups and should not be used for production environments. I recommend to use k8s in production.
+
+This setup is composed of 4 containers:
+- dex-worker
+- dex-overlord
+- postgres
+- example app
+
+Notes:
+- I needed to use some hacks to get the timing of all components right (https://github.com/docker/compose/issues/374). You will see `sh -c 'sleep x ;` to enforce the right order. Nevertheless it may happen that the startup fails for some components, since the execution time may depend on the host machine.
```

Comment:
That only works for dex-overlord. dex-worker will not start if the database is not populated. dexctl is not able to create the connectors and the example app will not work, since it requires client credentials